File: Iniciante/1049.js
let A1 = lines.shift();
let A2 = lines.shift();
let A3 = lines.shift();

if (A1 == 'vertebrado') {
  if (A2 == 'ave') {
    if (A3 == 'carnivoro') {
      console.log('aguia');
    } else if (A3 == 'onivoro') {
      console.log('pomba');
    }
  } else if (A2 == 'mamifero') {
    if (A3 == 'onivoro') {
      console.log('homem');
    } else if (A3 == 'herbivoro') {
      console.log('vaca');
    }
  }
} else {
  if (A2 == 'inseto') {
    if (A3 == 'hematofago') {
      console.log('pulga');
    } else if (A3 == 'herbivoro') {
      console.log('lagarta');
    }
  } else if (A2 == 'anelideo') {
    if (A3 == 'hematofago') {
      console.log('sanguessuga');
    } else if (A3 == 'onivoro') {
      console.log('minhoca');
    }
  }
}

File: Iniciante/1049.py
A1 = str(input())
A2 = str(input())
A3 = str(input())

if A1 == 'vertebrado':
  if A2 == 'ave':
    if A3 == 'carnivoro':
      print('aguia')
    elif A3 == 'onivoro':
      print('pomba')
  elif A2 == 'mamifero':
    if A3 == 'onivoro':
      print('homem')
    elif A3 == 'herbivoro':
      print('vaca');
else:
  if A2 == 'inseto':
    if A3 == 'hematofago':
      print('pulga')
    elif A3 == 'herbivoro':
      print('lagarta')
  elif A2 == 'anelideo':
    if A3 == 'hematofago':
      print('sanguessuga')
    elif A3 == 'onivoro':
      print('minhoca')
